tests/util.go

```diff
@@ -92,6 +102,11 @@ func CreateResourceWithFilePathTestNamespace(filePath string) {
 	execute(Result{cmd: "oc", verb: "create", filePath: filePath, nameSpace: NamespaceTestDefault})
 }
 
+func DeleteResourceWithNameTestNamespace(resourceType, resourceName string) {
```

Comment:
it would be a bit more general if the ns is also passed here. You could decide that if ns == "" to use the test default.